Tests/StreamChatUITests/SnapshotTests/ChatChannelList/ChatChannelListItemView_Tests.swift

```diff
@@ -766,13 +764,94 @@ final class ChatChannelListItemView_Tests: XCTestCase {
                 message: .mock(text: "Some text", createdAt: Date(timeIntervalSince1970: 1))
             )
         )
+        itemView.appearance.formatters.channelListMessageTimestamp = DefaultMessageTimestampFormatter()
+
+        XCTAssertEqual(
+            itemView.timestampText,
+            "12:00 AM"
+        )
+    }
+
+    func test_timestampText_whenCreatedAtIsToday_thenShowsTimeOnly() {
+        let channel: ChatChannel = .mock(
+            cid: .unique,
+            previewMessage: .mock(
+                createdAt: Date(timeIntervalSince1970: 1)
+            )
+        )
+
+        let mockCalendar = Calendar_Mock()
+        mockCalendar.mockIsDateInToday = true
+        let formatter = ChannelListMessageTimestampFormatter()
+        formatter.calendar = mockCalendar
+
+        let itemView = ChatChannelListItemView()
+        itemView.content = .init(channel: channel, currentUserId: nil)
+        itemView.appearance.formatters.channelListMessageTimestamp = formatter
 
         XCTAssertEqual(
             itemView.timestampText,
             "12:00 AM"
         )
     }
 
+    func test_timestampText_whenCreatedAtIsYesterday_thenShowsYesterday() {
```

Comment:
Shouldn't these tests be moved to a file that only tests `ChannelListMessageTimestampFormatter`?

Reply:
Yeah, I could do that, but there were no tests for any of the formatters, so I opted to do it here since it existed here already. In this case, an integration test here will work pretty much as the unit test 🤔 But I can move them to the formatter only as well

Reply:
It is ok for now, but we should ideally test things in as isolated environments as possible